docs/get-started/get-an-allocation.md

```diff
@@ -89,6 +93,20 @@ integer value that corresponds to multiple openstack resource quotas. For exampl
     Groups** are provided to each approved projects regardless of units of requested
     quota units.
 
+### Request A New OpenShift Resource Allocation for OpenShift project
+
+![Request A New OpenShift Resource Allocation](images/coldfront-request-new-openshift-allocation.png)
+
+!!! warning "Very Important"
```

Comment:
I suggest `:warning:` :warning: instead of `!!!` it catches the eye.

Reply:
Nevermind I see that your markdown to html changes !!! warning to something else.